pdfModel.cpp

```diff
@@ -107,10 +112,11 @@ int PdfModel::loadProvider()
 {
     DEBUG << "Loading image provider...";
     QQmlEngine *engine = QQmlEngine::contextForObject(this)->engine();
+    const QString& prefix = QString::number(quintptr(this));
+    providerName = prefix + "Poppler";
```

Comment:
I'd personally prefer `poppler<number>` prefix. Using uppercase letters in URI protocol part isn't a common thing.